README.md

```diff
@@ -15,44 +15,41 @@
 
 <h2 align="center">Installation</h2>
 
-```
-npm i -g bhailang
+```bash
+$ npm i -g bhailang
```

Comment:
If someone will copy the code, it will also copy that dollar sign. Can we simply have the previous version of this?

Reply:
yes makes sense, the reason i used itw as to show that it's a terminal script, I will just update the heading to do it ig